amphora-image-builder/elements/vm_baseline/install.d/90-run-vm_baseline

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#install ansible
+/usr/bin/python3 -m pip install ansible
```

Comment:
Could we use a venv, just because this will break in 22.04 onwards and it's bad to potentially trash the system packages anyway:

```
python3 -m venv /tmp/amphora-venv
source /tmp/amphora-venv/bin/activate
```

Reply:
Yup, done